Podfile.lock

```diff
@@ -2,14 +2,14 @@ PODS:
   - ACKLocalization (0.3.3)
   - LicensePlist (2.16.0)
   - Smartling.i18n (1.0.14)
-  - SwiftGen (6.2.0)
-  - SwiftLint (0.39.2)
+  - SwiftGen (6.3.0)
+  - SwiftLint (0.40.3)
 
 DEPENDENCIES:
   - ACKLocalization (~> 0.3)
   - LicensePlist (~> 2.14)
   - Smartling.i18n (~> 1.0)
```

Comment:
❓ I think we can remove Smartling since SwiftGen is updated to 6.3. Should we do that in separate PR?

Reply:
This will be separate MR considering #56.